src/kitFilterHelpers.ts

```diff
@@ -0,0 +1,56 @@
+import { generateHash } from "./utils";
```

Comment:
Should we consider moving `generateHash` out of utils and into this class?

Reply:
`getRampNumber` which is in Utils also uses `generateHash`, so it would be awkward to import a class into utils file.